templates/core/sitemap.xml

```diff
@@ -2,7 +2,7 @@
 <urlset xmlns="http://www.sitemaps.org/schemas/sitemap/0.9">
     {% for release in releases -%}
         <url>
-            <loc>https://docs.rs/{{ release[0] }}</loc>
+            <loc>https://docs.rs/{{ release[0] }}/latest/{{ release[0] }}</loc>
```

Comment:
This is not correct. The crate name can be different from the package name (e.g. dashes vs underscores). Unlike the main /:package url, we don't autocorrect typos here (and in general the two can be completely unrelated; take a look at `rustc-workspacd-core`).

Not sure if the template already has access to the crate name, you'll need to add it if not.

Reply:
Thanks for catching that. I've updated the PR with what I think fixes it. Having trouble testing because I am currently getting an error from the rustwide builder:

```
cargo run -- build crate cairo-rs 0.15.12
...
2022/07/12 09:52:15 [INFO] rustwide::cmd: [stderr] warning: spurious network error (1 tries remaining): [6] Couldn't resolve host name (Could not resolve host: crates.io)
2022/07/12 09:52:15 [INFO] rustwide::cmd: [stderr] warning: spurious network error (1 tries remaining): [6] Couldn't resolve host name (Could not resolve host: crates.io)
2022/07/12 09:52:15 [INFO] rustwide::cmd: [stderr] warning: spurious network error (1 tries remaining): [6] Couldn't resolve host name (Could not resolve host: crates.io)
2022/07/12 09:52:15 [INFO] rustwide::cmd: [stderr] error: failed to download from `https://crates.io/api/v1/crates/anyhow/1.0.58/download`
2022/07/12 09:52:15 [INFO] rustwide::cmd: [stderr] 
2022/07/12 09:52:15 [INFO] rustwide::cmd: [stderr] Caused by:
2022/07/12 09:52:15 [INFO] rustwide::cmd: [stderr]   [6] Couldn't resolve host name (Could not resolve host: crates.io)
...
```

Reply:
I think that's an error from the build script itself? hard to tell from the snippet you posted. try building some other crate without a build script, like `regex_syntax`.